### DISCUSSION TITLE
Samba Mac

### DISCUSSION THREAD

**Discussion by FirestoneXD** [AlexGustafsson/homebridge-wol#233] [Questions]
### Backstory

Hello everyone, I've only recently started using Homebridge and now I want to control my PC via Homekit. I installed Homebridge on an imac (High Sierra). Unfortunately there is no Samba for the imac, is there any other way I can do it?

![go](https://user-images.githubusercontent.com/123833213/215589709-d9e5a8cc-141e-41ce-9050-1000be99f9b2.png)
![go2](https://user-images.githubusercontent.com/123833213/215591478-287623be-f554-452c-83fc-66fa13868acc.png)

### Issue

He always gave me the error Out: (Not Super-User) by trying to shutdown. It´s not possible to start either
regreet Luca

### Environment

```Markdown
- os and os version: macOS High Sierra
- node version:v17.9.1
- homebridge version:v1.6.0
- homebridge-wol version:v5.0.2
```

### Configuration

![go2](https://user-images.githubusercontent.com/123833213/215591502-ec73d130-1c7a-4aba-a4b1-f846d5908def.png)

### Logs

```Markdown
When I follow these steps:
1. Make sure `logLevel` is set to `Debug` on all `NetworkDevice` accessories
2. Run `homebridge`
3. Let it run for a couple of minutes
4. Try to turn a device on or off using the plugin
5. Wait one minute
6. Repeat step 4

I get the following log:

Please paste your log here
```
```


### Notes

_No response_

**Comment by AlexGustafsson** [comment]
You can still install the samba CLI utilities to enable your Mac to turn off Windows PCs. See: https://superuser.com/questions/878070/how-to-shutdown-a-windows-computer-via-a-mac-terminal. Closing.
